components/hab/src/command/plan/init.rs

```diff
@@ -115,10 +117,40 @@ pub fn start(ui: &mut UI,
                your habitat. There are several hooks to create and tweak! See the full list \
                with info here: https://www.habitat.sh/docs/reference/plan-syntax/#hooks"));
 
+    try!(render_gitignore(ui, &root));
+
     try!(ui.end("A happy abode for your code has been initialized! Now it's time to explore!"));
     Ok(())
 }
 
+fn render_gitignore(ui: &mut UI, root: &str) -> Result<()> {
+    let target_path = format!("{}/../.gitignore", root);
+    let path = Path::new(&target_path);
+
+    match path.exists() {
```

Comment:
I think so too. I was following other examples in this file, but I like that better also.